### PR TITLE
HAWQ-672. Add python module pygresql back into hawq workspace

### PR DESCRIPTION
**Pull request by paul-guo-**
In this patch, all of the new added files come from the workspace after reverting HAWQ-271. Other changes mostly follow the original patch for HAWQ-271.

### CODE REVIEW

File: tools/bin/generate-greenplum-path.sh [hunk 1]
```diff
@@ -90,16 +90,9 @@ EOF
...
 
 #setup PYTHONPATH
-# OSX does NOT need pygresql/ path
-if [ "${PLAT}" = "Darwin" ] ; then
-cat <<EOF
-PYTHONPATH=\$GPHOME/lib/python:\$PYTHONPATH
-EOF
-else
 cat <<EOF
 PYTHONPATH=\$GPHOME/lib/python:\$GPHOME/lib/python/pygresql:\$PYTHONPATH
```
Comment (radarwave) [inline tools/bin/generate-greenplum-path.sh]: Please check if we need '$GPHOME/lib/python/pygresql' in $PYTHONPATH ?

File: tools/bin/ext/__init__.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+from tokens import *
+from events import *
+from nodes import *
+
+from loader import *
+from dumper import *
+
+try:
+    from cyaml import *
```
Comment (radarwave) [inline tools/bin/ext/__init__.py]: We would better remove the codes not related to pygresql in this '__init__.py' file.

**Comment by radarwave** [comment]
Looks good to me. +1

**Comment by yaoj2** [comment]
＋1

**Pull request closed without merging**